### ISSUE TITLE
EMF application model validation failed

### ISSUE THREAD

**Issue by lacherp** [opcoach/datasample#6]
Diagnostic ERROR source=org.eclipse.emf.ecore code=0 Diagnosis of org.eclipse.e4.ui.model.fragment.impl.StringModelFragmentImpl@19a64eae{platform:/plugin/com.opcoach.datasample.template.generator.ui/fragment.e4xmi#_vmvQUAy2EeuHLYVCt3xCyw} data=[org.eclipse.e4.ui.model.fragment.impl.StringModelFragmentImpl@19a64eae (featurename: handlers, parentElementId: xpath:/, positionInList: null)] [Diagnostic ERROR source=org.eclipse.emf.ecore code=1 The required feature 'command' of 'org.eclipse.e4.ui.model.application.commands.impl.HandlerImpl@7fb66650{platform:/plugin/com.opcoach.datasample.template.generator.ui/fragment.e4xmi#_wbOoUAy2EeuHLYVCt3xCyw}' must be set data=[com.opcoach.datasample.template.generator.ui.handler.0=org.eclipse.e4.ui.model.application.commands.impl.HandlerImpl@7fb66650 (tags: null, contributorURI: null) (contributionURI: bundleclass://com.opcoach.datasample.template.generator.ui/com.opcoach.datasample.template.generator.ui.GenerateDataSampleTemplateHandler, object: null), org.eclipse.emf.ecore.impl.EReferenceImpl@4f966719 (name: command) (ordered: true, unique: true, lowerBound: 1, upperBound: 1) (changeable: true, volatile: false, transient: false, defaultValueLiteral: null, unsettable: false, derived: false) (containment: false, resolveProxies: true)]]

proposal fix: append command definition  in fragment model ( com.opcoach.datasample.template.generator.ui ) :  

![image](https://user-images.githubusercontent.com/58602053/99699943-e0cc2880-2a92-11eb-95f8-6e2892752338.png)

**Comment by opcoach** [comment]
I will check that... you mean there was an error on loading because the command was not defined in the fragment but expected in the main E4 application ? May be only importing it would be enough ?

**Comment by lacherp** [comment]
yes of course we can import it if you have defined it elsewhere, the error log comes mainly  because there is no command id associated with handler
